### PR TITLE
Document storage configuration options

### PR DESCRIPTION
**Pull request by pmeulen**
Add / improve documentation of configuration option of the user-, usersecret-, and session storage.

Describe how the probabilistic cleanup of the PDO state storage works and include creating an index on the expire column

### CODE REVIEW

File: library/tiqr/Tiqr/UserSecretStorage/Pdo.php [hunk 1]
```diff
@@ -42,6 +42,17 @@
     userid varchar(30) NOT NULL UNIQUE,
     secret varchar(128),
 );
+
+ * @see Tiqr_UserSecretStorage::getSecretStorage()
+ * @see Tiqr_UserSecretStorage_Interface
+ *
+ * Supported options:
+ * path : Path to the directory where the user data is stored
+ * Supported options:
```
Comment (MKodde) [inline library/tiqr/Tiqr/UserSecretStorage/Pdo.php:51]: Nitpick: Duplicate `Supported options:`

File: library/tiqr/Tiqr/StateStorage/Pdo.php [hunk 2]
```diff
@@ -20,13 +20,59 @@
...
+ * Add to that the keys created by the application that uses the library (e.g. Stepup-Tiqr creates one key).
+ *
+ * A good starting point is using the typical peak number of authentications per hour and setting the cleanup_probability
+ * so that it would on average run once during such an hour. Since the number of authentications >> number of enrollments
+ * A good starting value for cleanup_probability would be:
+ *
+ *  1 / ( [number of authentications per hour] * ( 2 + [ keys used by application] ) )
+ *
+ * E.g. for 10000 authentications per hour, with one key created by the application:
+ *      cleanup_probability = 1 / (10000 (2 + 1)) = 0.00003
```
Comment (MKodde) [inline library/tiqr/Tiqr/StateStorage/Pdo.php:50]: :ok_hand:

**Review by MKodde — APPROVED** [review]
Great documentation additions!

**Comment by pmeulen** [comment]
Thx, fixed the copy/paste error

**Pull request merged**